pkg/apis/kubedirector.bluedata.io/v1alpha1/zz_generated.openapi.go

```diff
@@ -0,0 +1,227 @@
+	}
+}
+
+func schema_pkg_apis_kubedirector_v1alpha1_KubeDirectorAppSpec(ref common.ReferenceCallback) common.OpenAPIDefinition {
+	return common.OpenAPIDefinition{
+		Schema: spec.Schema{
+			SchemaProps: spec.SchemaProps{
+				Description: "KubeDirectorAppSpec defines the desired state of KubeDirectorApp",
+				Properties:  map[string]spec.Schema{},
+			},
+		},
+		Dependencies: []string{},
+	}
+}
+
+func schema_pkg_apis_kubedirector_v1alpha1_KubeDirectorAppStatus(ref common.ReferenceCallback) common.OpenAPIDefinition {
+	return common.OpenAPIDefinition{
+		Schema: spec.Schema{
+			SchemaProps: spec.SchemaProps{
+				Description: "KubeDirectorAppStatus defines the observed state of KubeDirectorApp",
+				Properties:  map[string]spec.Schema{},
+			},
+		},
+		Dependencies: []string{},
+	}
+}
+
+func schema_pkg_apis_kubedirector_v1alpha1_KubeDirectorCluster(ref common.ReferenceCallback) common.OpenAPIDefinition {
+	return common.OpenAPIDefinition{
+		Schema: spec.Schema{
+			SchemaProps: spec.SchemaProps{
+				Description: "KubeDirectorCluster is the Schema for the kubedirectorclusters API",
+				Properties: map[string]spec.Schema{
+					"kind": {
+						SchemaProps: spec.SchemaProps{
+							Description: "Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds",
+							Type:        []string{"string"},
+							Format:      "",
+						},
+					},
+					"apiVersion": {
+						SchemaProps: spec.SchemaProps{
+							Description: "APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources",
+							Type:        []string{"string"},
+							Format:      "",
+						},
+					},
+					"metadata": {
+						SchemaProps: spec.SchemaProps{
+							Ref: ref("k8s.io/apimachinery/pkg/apis/meta/v1.ObjectMeta"),
+						},
+					},
+					"spec": {
+						SchemaProps: spec.SchemaProps{
+							Ref: ref("github.com/bluek8s/kubedirector/pkg/apis/kubedirector.bluedata.io/v1alpha1.KubeDirectorClusterSpec"),
+						},
+					},
+					"status": {
+						SchemaProps: spec.SchemaProps{
+							Ref: ref("github.com/bluek8s/kubedirector/pkg/apis/kubedirector.bluedata.io/v1alpha1.KubeDirectorClusterStatus"),
+						},
+					},
+				},
+			},
+		},
+		Dependencies: []string{
+			"github.com/bluek8s/kubedirector/pkg/apis/kubedirector.bluedata.io/v1alpha1.KubeDirectorClusterSpec", "github.com/bluek8s/kubedirector/pkg/apis/kubedirector.bluedata.io/v1alpha1.KubeDirectorClusterStatus", "k8s.io/apimachinery/pkg/apis/meta/v1.ObjectMeta"},
```

Comment:
Can we use a constant for this. Used at several places.

github.com/bluek8s/kubedirector/pkg/apis/kubedirector.bluedata.io

Makes it easier when moving to kubedirectorplus

Reply:
Since this is generated code I think this particular spot is out of our hands. I'll talk to you more about this offline.